### ISSUE TITLE
Criar visualização Google Analytics no Tableau Online

### ISSUE THREAD

**Issue by dossanluc** [bireme/data-governance#222]
Visualização por artigo (filtro principal id) (regional)
	Range de data (30 dd, 60 dd, 180 dd, 365 dd)

**Comment by dossanluc** [comment]
Primeira visualização disponível no Tableau Online, estamos testando o agendamento da extração do Google Analytics.

**Comment by dossanluc** [comment]
Esta disponível uma nova visualização, que permite o acesso com o parâmetro ID da publicação, abaixo estão alguns exemplos: 
- https://prod-useast-a.online.tableau.com/#/site/biremeopsoms/views/IAHX_PageViews_2/Info?pub=bde-12571
- https://prod-useast-a.online.tableau.com/#/site/biremeopsoms/views/IAHX_PageViews_2/Info?pub=abr-387
- https://prod-useast-a.online.tableau.com/#/site/biremeopsoms/views/IAHX_PageViews_2/Info?pub=mdl-31829464
- https://prod-useast-a.online.tableau.com/#/site/biremeopsoms/views/IAHX_PageViews_2/Info?pub=biblio-893733

**Comment by renatomurasaki** [comment]
@dossanluc qual é o período de estatísticas que temos no momento?

**Comment by dossanluc** [comment]
@renatomurasaki essa é a base de 2019, estou trabalhando na base de 2020, para ter os mesmo acesso.

**Comment by renatomurasaki** [comment]
@dossanluc repense, por favor, a forma de visualizar as cidades/estados e os meses. Veja esse exemplo:

https://prod-useast-a.online.tableau.com/#/site/biremeopsoms/views/IAHX_PageViews_2/Info?pub=biblio-848534&:iid=2

Acho que poderia plotar as cidades no próprio mapa. Meses seria melhor um gráfico.

**Comment by dossanluc** [comment]
Sim, já estava finalizando a atualizei, acebei de publicar. Obrigado!

**Comment by dossanluc** [comment]
@renatomurasaki, realizei uma nova alteração conforme solicitado (Acho que poderia plotar as cidades no próprio mapa. Meses seria melhor um gráfico.), a mesma já esta disponpivel.

**Comment by dossanluc** [comment]
Visualização para estatística por publicação atualizada, conforme itens abaixo:
- Filtro por ano
- Mostrar anos no page / month
- Mudar o titulo para cima
- Logo do lado esquerdo
- Lista de Pais (linkar com todas as planilhas)
- Map - No detalhe City,Pai (Sem lable), Indicadores com Lable
- Link para voltar para publicação original

Pendencias
- 4 Idiomas, um dashboard para cada idioma
- Visualização com join de 2 bases
- Tornar o acesso tableu online publico

**Comment by dossanluc** [comment]
Visualização para estatística por publicação atualizada, conforme itens abaixo:
- Filtro por ano
- Mostrar anos no page / month
- Mudar o titulo para cima
- Logo do lado esquerdo
- Lista de Pais (linkar com todas as planilhas)
- Map - No detalhe City,Pai (Sem lable), Indicadores com Lable
- Link para voltar para publicação original
- 4 Idiomas, um dashboard para cada idioma
- Visualização com join de 2 bases

Pendencias
- Tornar o acesso tableau online publico
